src/components/Dashboard/Users.js

```diff
@@ -7,63 +7,139 @@ import {
   DashboardLayout as Layout
 } from 'components/Dashboard/';
 import { bindActionCreators } from 'redux';
-import { Table } from 'react-bootstrap';
+import { Col, Container, Row, Table } from 'react-bootstrap';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import { actions as usersActions } from 'reducers/users';
-import { getAllUsers } from 'selectors/users';
+import { getAllUsers, getUsersPager } from 'selectors/users';
 
 export class Users extends Component {
   static propTypes = {
     actions: PropTypes.object.isRequired,
-    users: PropTypes.array,
-    layoutOptions: PropTypes.object
+    layoutOptions: PropTypes.object,
+    collection: PropTypes.array,
+    pager: PropTypes.object
   };
 
+  constructor(props) {
+    super(props);
+    this.state = { limit: 20, page: 1 };
```

Comment:
These numbers could be defined as instance members like `this.defaultState = { limit: 20, page: 1};` and then you can re-use it in `componentDidMount()`

Reply:
Updated in Pagination HOC